main.js

```diff
@@ -1,9 +1,10 @@
 // this should only include code for event listeners/handlers.
 
 let startButton = $('.js-start-button');
+let gameBoard = $('.game-board');
```

Comment:
A common convention you'll see for variables that represent jQuery elements is that they'll be named with a $ in front of them, like `let $gameBoard` or `let $startButton` -- that just helps other developers recognize really easily that a variable represents some sort of DOM element that was selected with jQuery